wdio.conf.js

```diff
@@ -34,6 +34,9 @@ exports.config = {
         ],
         products: [
             './test/specs/products.js'
+        ],
+        productDetailed: [
+            './test/specs/productDetailed.js'
```

Comment:
why productDetailed and not productDetails?

Reply:
correction made